<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>polyCountBudget</class>
 <widget class="QWidget" name="polyCountBudget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Poly Count Budget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <widget class="QGroupBox" name="polyCountBudgetGroupBox">
     <property name="title">
      <string>Enter Poly Count</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QRadioButton" name="triangleRadio">
          <property name="text">
           <string>Triangle</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="vertexRadio">
          <property name="text">
           <string>Vertex</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="fromLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;From:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="startBudgetSpinBox">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="maximum">
           <number>9999999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;TO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="endBudgetSpinBox">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="maximum">
           <number>9999999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="tagLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Tag: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="tagComboBox">
          <item>
           <property name="text">
            <string>Extremely low</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Low</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Medium</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>High</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Very high</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Extremely high</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add Budget</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetButton">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>endBudgetSpinBox</zorder>
     <zorder>tagComboBox</zorder>
     <zorder>tagLabel</zorder>
     <zorder>resetButton</zorder>
     <zorder>addButton</zorder>
     <zorder></zorder>
     <zorder></zorder>
     <zorder></zorder>
     <zorder>label</zorder>
     <zorder></zorder>
     <zorder>fromLabel</zorder>
     <zorder>endBudgetSpinBox</zorder>
     <zorder>triangleRadio</zorder>
     <zorder>vertexRadio</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="budgetGroupBox">
     <property name="title">
      <string>Budget</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QTableView" name="budgetTableView">
        <property name="gridStyle">
         <enum>Qt::DashLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>endBudgetSpinBox</zorder>
  <zorder>budgetLabel</zorder>
  <zorder>tagLabel</zorder>
  <zorder>tagComboBox</zorder>
  <zorder>addButton</zorder>
  <zorder>resetButton</zorder>
  <zorder>polyCountBudgetGroupBox</zorder>
  <zorder>budgetGroupBox</zorder>
  <zorder>budgetGroupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
